reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -82,7 +82,14 @@
   "scan_results" : [ {
     "_id" : 0,
     "provenance" : {
-      "download_time" : "1970-01-01T00:00:00Z"
+      "download_time" : "1970-01-01T00:00:00Z",
+      "vcs_info" : {
+        "type" : "Git",
+        "url" : "https://github.com/oss-review-toolkit/ort.git",
+        "revision" : "master",
+        "resolved_revision" : "master",
```

Comment:
Am I right that the `resolved_revision` in non-synthetic scan-results can only ever by a `SHA1`, but never a branch or tag name? If so, shall we use a SHA1 here (and below) as well?

I believe that'd be beneficial also because PackageConfigurations should be matched on `resolved_revision`, but not on `revision`. Using same value for both makes the test also less strict.

Reply:
Done.